modules/apps/frontend-js/frontend-js-react-web/src/main/resources/META-INF/resources/js/hooks/useThunk.ts

```diff
@@ -23,16 +23,16 @@ const {useRef} = React;
  * thunks (ie. functions that dispatch actions) as well as plain actions (ie.
  * objects).
  */
-export default function useThunk([state, dispatch]) {
+export default function useThunk([state, dispatch]: [any, any]) {
```

Comment:
Ah forgot to comment here. I intended to clean up these ugly `any` types and forgot about them.